pkg/security/tests/testopts.go

```diff
@@ -57,6 +57,7 @@ type testOpts struct {
 	tagsResolver                               tags.Resolver
 	snapshotRuleMatchHandler                   func(*testModule, *model.Event, *rules.Rule)
 	enableFIM                                  bool // only valid on windows
+	networkIngressEnabled                      bool
```

Comment:
The new option should probably be added to the Equal function
 https://github.com/DataDog/datadog-agent/blob/42a0692958957d71e9d044a4c4f1680907a118ae/pkg/security/tests/testopts.go#L101

Reply:
Nice catch, updating !